dpnp/dpnp_iface_manipulation.py

```diff
@@ -1414,20 +1506,37 @@ def unique(ar, **kwargs):
     return call_origin(numpy.unique, ar, **kwargs)
 
 
-def vstack(tup):
+def vstack(tup, *, dtype=None, casting="same_kind"):
     """
     Stack arrays in sequence vertically (row wise).
 
     For full documentation refer to :obj:`numpy.vstack`.
 
-    """
+    Examples
```

Comment:
Could you please add "Limitations", "Returns", and "See Also" sections similar to `hstack`?